index.html

```diff
@@ -643,24 +643,31 @@ <h2>Security</h2>
           </ul>
         </div>
         <p><span class="rfc2119-assertion" id="common-constraints-security-2">
-            Conformant Consumers MUST support all of these security schemes.</span>
+            Conformant Consumers MUST support at least all of these security schemes.</span>
         </p>
-	    
+
         <p><span class="rfc2119-assertion" id="common-constraints-security-3">
-            A Thing MAY implement multiple security schemes.</span>
+            A Thing MAY implement multiple security schemes, at least one of the above MUST be supported.</span>
```

Comment:
As I understand it the tooling for extracting assertions doesn't support multiple RFC 2119 key words in a single assertion. You probably need to split this into two separate assertions with their own id.